bolt/metrics.go

```diff
@@ -209,12 +223,15 @@ func (c *Client) Collect(ch chan<- prometheus.Metric) {
 
 	orgs, buckets, users, tokens := 0, 0, 0, 0
 	dashboards, scrapers, telegrafs := 0, 0, 0
+	remotes, replications := 0, 0
 	_ = c.db.View(func(tx *bolt.Tx) error {
 		buckets = tx.Bucket(bucketBucket).Stats().KeyN
 		dashboards = tx.Bucket(dashboardBucket).Stats().KeyN
 		orgs = tx.Bucket(organizationBucket).Stats().KeyN
 		scrapers = tx.Bucket(scraperBucket).Stats().KeyN
 		telegrafs = tx.Bucket(telegrafBucket).Stats().KeyN
+		remotes = tx.Bucket(remoteBucket).Stats().KeyN
```

Comment:
Do we care about whether `View` fails?  Our function can't fail, but the BoltDB itself can fail, and we should probably handle that.  Currently, we ignore any error from `Collect`, but at the least, we could return the error up the stack in anticipation of being more careful in a future refactor of this code.

Reply:
Looks like the `Collect` method is on an interface in Prometheus that does not have an error return. So I'm not sure there's much we can do to pass this up the stack in its current form. Notably, all the `MustNewConstMetric` calls that follow this will panic if they encounter an error.

Reply:
If the `MustNew...` are panicking on error, then maybe we should panic if `View` fails as well. Not my favorite error handling strategy, but we don't seem to have a log object, and we have nothing paying attention up the stack if we can't talk to BoltDB (which is why we would fail here).  The function we pass in can't fail, but all the transaction `begin` and `rollback` can.

Not a blocker for the release, but I hate failures that are swallowed. They just turn up later without any context information.....

Reply:
Yeah I don't love adding a new panic, but you're definitely correct that there is really no other easily accessible reporting mechanism at the moment if something goes wrong

Reply:
Well, let's leave it for the next round....